rhaptos/cnxmlutils/xsl/cnxml-to-html5.xsl

```diff
@@ -27,11 +29,11 @@
 </xsl:template>
 
 <xsl:template match="c:div">
-  <div><xsl:apply-templates select="@*|node()"/></div>
+  <div><xsl:apply-templates select="@*|node()"/><xsl:comment/></div>
```

Comment:
Can the comments only be added when there are no child nodes?

Reply:
In my tests empty comments are not visible on the XSLT processor xsltproc / libxslt. Result never included `<!-- -->`.

See also
https://github.com/openstax/rhaptos.cnxmlutils/pull/179#issuecomment-498396237

We will test/verify this behaviour also on qa.

It would be redundant or only a safety net (for another XSLT processor/version) to write a function on empty comments.